nixos/modules/hardware/all-firmware.nix

```diff
@@ -49,7 +49,7 @@ in {
         rt5677-firmware
         rtl8723bs-firmware
         rtl8761b-firmware
-        rtlwifi_new-firmware
+        rtw88-firmware
```

Comment:
How would that break eval, given t here is an alias for the former?

Reply:
I'm guessing this only happens when setting `nixpkgs.config.allowAliases = false;`.

Reply:
Yep, that's probably my exact problem :/ Sorry for making the problem seem bigger than it actually is